rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
Can we at least have the option of setting alpha, even if we don't use it currently?